README.md

```diff
@@ -2,18 +2,115 @@
 
 Sym's serverless integrations let you use Sym workflows to manage resources that the Sym platform does not directly integrate with. Sym invokes your custom function with the appropriate metadata and then your function implementation updates your backend systems.
 
-These integrations can also be used as an "escape hatch" to trigger escalations in services Sym does not yet support.
+## Schema
 
-There are two serverless integrations supported today:
+The schema of Sym's requests to you can be found [here](https://sym.stoplight.io/docs/sym-reporting), with example payloads in the [`test`](test) directory.
 
-- [HTTP](https://docs.symops.com/docs/http)
-- [AWS Lambda](https://docs.symops.com/docs/aws-lambda)
+## Deploying
 
-The templates in this repo will be focused on deploying AWS Lambdas, but the code can easily be reused to parse inbound Sym requests from an HTTP server of your choosing.
+Each template includes a `build.sh` that you can use to package your AWS Lambda function for deployment. Once your function is packaged, you can upload it to S3 or use the `update-function-code` API:
 
-## Schema
+```
+$ aws lambda update-function-code \
+  --function-name ${FUNCTION_NAME} \
+  --zip-file fileb://handler.zip
+```
+
+## Python
+
+### Lambda Configuration
+
+You can try the template out by starting with this Terraform snippet or the equivalent in another deployment tool:
+
+```
+resource "aws_lambda_function" "this" {
+  ...
+
+  s3_bucket = "sym-releases"
+  s3_key = "lambda-templates/python.zip"
+  handler = "handler.handle"
+  runtime = "python3.9"
+
+  ...
+}
+```
+
+### Local Testing
+
+[`handler.py`](python/handler.py) lets you run the handler locally using:
+
+```
+$ pip install -r requirements.txt
+$ python handler.py [-e | -d]
```

Comment:
for here and also the ts/go sections, it might be helpful to document here that `-e` is to test handling an escalate event while `-d` is de-escalate